tests/datasets/test_in_context_learning_datasets.py

```diff
@@ -0,0 +1,118 @@
+# Copyright 2022 MosaicML Composer authors
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+
+import pytest
+from torch.utils.data import DataLoader
+
+from composer.core import Evaluator
+from composer.datasets.in_context_learning_evaluation import get_lm_task_dataloader, get_mc_task_dataloader
+from composer.loggers import InMemoryLogger
+from composer.models.gpt2 import create_gpt2
+from composer.trainer import Trainer
+from tests.common import device, world_size
+
+
+@pytest.mark.parametrize('dataset_uri', ['lambada_small.jsonl'])
+def test_lm_task_dataloader(dataset_uri, tiny_gpt2_tokenizer):
+    local_data = os.path.join(os.path.dirname(__file__), 'local_data')
+
+    tokenizer = tiny_gpt2_tokenizer
+    dataset_uri = f'{local_data}/{dataset_uri}'
+    dl = get_lm_task_dataloader(dataset_uri,
+                                tokenizer,
+                                2,
+                                max_seq_len=2048,
+                                eos_tok_id=tokenizer.eos_token_id,
+                                num_fewshot=1,
+                                preamble_string='',
+                                example_delimiter='\n',
+                                continuation_delimiter='')
+
+    assert isinstance(dl.dataloader, DataLoader)  # pyright
```

Comment:
I think it may be worth it to actually write out the expected value of each of these keys for both of these dataloader tests, for the first example. otherwise they are just testing that code doesn't crash and that the dictionary returned is correctly formed. As the dataset got more complicated with fewshot, I think this became more necessary.